blocks/subscriptions-block/components/useCartOrderDetail.test.jsx

```diff
@@ -0,0 +1,5 @@
+// cart = true
+// cart = false, cartLocalStorage = false, pendingOrder = false
+// cart = false, cartLocalStorage = false,  pendingOrder = true
+// cart = false, cartLocalStorage = true, pendingOrder = false
+// cart= false, cartLocalStorage = true, pendingOrder = true 
```

Comment:
Assuming this is a set of comments reminding someone else what tests are needed?  If not, we can remove this file until the tests are written (which I assume there is a ticket for?  Testing at some point will be recovering to the 80%+ area for coverage and I don't know if anyone can easily identify all the branches of that promise set.

I know this is "rushed" so if you have a ticket for test coverage (ie: scheduled work) then I can go ahead with approval on this (related) file.